iidm/iidm-api/src/main/java/com/powsybl/iidm/network/LoadingLimitsAdder.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
```

Comment:
Fix copyright information. I think you rename the CurrentLimitsAdder to LoadingLimitsAdder, but maybe you should fix it